Dockerfile.deps

```diff
@@ -15,23 +15,45 @@ RUN sh -c '[ -n "$NO_LIQUID" ] && mkdir -p /wally/wally_dist || ( \
 FROM debian:bullseye@sha256:4d6ab716de467aad58e91b1b720f0badd7478847ec7a18f66027d0f8a329a43c
 SHELL ["/bin/bash", "-c"]
 
+ENV CORE_BUILD_DEPS="autoconf autotools-dev automake libtool pkg-config bsdmainutils build-essential"
+ENV ESPLORA_BUILD_DEPS="clang cmake curl git"
 RUN mkdir -p /srv/explorer \
  && apt-get -yqq update \
  && apt-get -yqq upgrade \
- && apt-get -yqq install clang cmake curl git tor
+ && apt-get -yqq install ${ESPLORA_BUILD_DEPS} tor ${CORE_BUILD_DEPS}
+
 
 RUN git clone --quiet --depth 1 --single-branch --branch v0.39.0 https://github.com/nvm-sh/nvm.git /root/.nvm \
  && rm -rf /root/.nvm/.git \
  && source /root/.nvm/nvm.sh \
  && nvm install v17.1.0
 
-ENV SHA256SUM_BITCOINCORE=59ebd25dd82a51638b7a6bb914586201e67db67b919b2a1ff08925a7936d1b16
-ENV VERSION_BITCOINCORE=22.0
-RUN curl -sL -o bitcoin.tar.gz "https://bitcoincore.org/bin/bitcoin-core-${VERSION_BITCOINCORE}/bitcoin-${VERSION_BITCOINCORE}-x86_64-linux-gnu.tar.gz" \
- && echo "${SHA256SUM_BITCOINCORE}  bitcoin.tar.gz" | sha256sum --check \
- && tar xzf bitcoin.tar.gz -C /srv/explorer \
- && ln -s "/srv/explorer/bitcoin-${VERSION_BITCOINCORE}" /srv/explorer/bitcoin \
- && rm bitcoin.tar.gz
+# Build core from sources until PR https://github.com/bitcoin/bitcoin/pull/23387 is merged
+ENV CORE_PATCH=contrib/0001-add-support-to-save-fee-estimates-without-shutting-d.patch
+ENV CORE_SRC=/root/bitcoin
+COPY ${CORE_PATCH} /${CORE_PATCH}
+RUN git clone --quiet --depth 1 --branch v22.0 --single-branch --recursive https://github.com/bitcoin/bitcoin.git ${CORE_SRC} \
```

Comment:
It would be faster to download the [tag tarball from github](https://github.com/bitcoin/bitcoin/archive/refs/tags/v22.0.tar.gz) instead of using `git clone`. But speed isn't critical here so its also fine to keep as is.

Reply:
true but then you'd need to apply the patch with patch or something like that